lib/devise_security_extension/models/password_archivable.rb

```diff
@@ -16,7 +16,7 @@ def validate_password_archive
 
       # validate is the password used in the past
       def password_archive_included?
-        unless deny_old_passwords.is_a? Fixnum
+        unless deny_old_passwords.is_a? 1.class
```

Comment:
Yup, there's just an `Integer` class now http://blog.bigbinary.com/2016/11/18/ruby-2-4-unifies-fixnum-and-bignum-into-integer.html